demo/qemu/README.md

```diff
@@ -0,0 +1,369 @@
+<p align="center">
+  <img alt="Network" src="demo-qemu.svg" />
+</p>
+
+## Dependencies
+Hardware:
+- CPU with 4 cores
+- 8GB RAM
+
+Software:
+- OS: Ubuntu 18.04 or newer
+- [docker](https://www.docker.com/)
+- [QEMU](https://www.qemu.org/)
+- iproute2
+
+## Build virtual machine images
+Use [d2vm](https://github.com/linka-cloud/d2vm) for build virtual machine image from Dockerfile.
+
+Prepare environments:
+```
+$ sudo apt install docker.io qemu-system-x86 iproute2
+$ docker pull linkacloud/d2vm:v0.2.0-1934915
+$ alias d2vm="docker run --rm -i -t --privileged -v /var/run/docker.sock:/var/run/docker.sock -v $PWD:/build -w /build linkacloud/d2vm:v0.2.0-1934915"
+$ cd yanet/demo/qemu
+$ mkdir build_vm
+```
+
+### Host A
+```
+$ d2vm build -p root -f host_a/Dockerfile -o build_vm/host_a.qcow2 --force host_a
+```
+- `-p root` - root user password.
+- `-f host_a/Dockerfile` - name of the Dockerfile.
+- `-o build_vm/host_a.qcow2` - the output image.
+- `--force` - override output qcow2 image.
+
+### Host B
+```
+$ d2vm build -p root -f host_b/Dockerfile -o build_vm/host_b.qcow2 --force host_b
+```
+
+### YANET
+Build docker image with `YANET`:
+```
+$ cd yanet
+$ docker build -f yanet.Dockerfile -t yanetplatform/yanet .
```

Comment:
Oh, I realized that `cd yanet` is changing working directory to the root of the repository. I thought it would be going to `demo/qemu/yanet`.
Now I have found yanet.Dockerfile and I will try to continue the instruction.